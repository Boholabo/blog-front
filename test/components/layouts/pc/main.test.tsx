import React from 'react';
import { render } from '../../../testUtils';
import PcMain from '../../../../components/layouts/pc/main';

describe('layouts/pc/main component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <PcMain>
        <div>test</div>
      </PcMain>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
