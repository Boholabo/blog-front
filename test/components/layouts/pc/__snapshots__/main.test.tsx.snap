// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layouts/pc/main component matches snapshot 1`] = `
<DocumentFragment>
  <div
    id="app-main"
  >
    <main
      class="main-container"
      id="main"
    >
      <div
        class="main-content"
        id="main-content"
      >
        <div>
          test
        </div>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
